lib/utils/error-message.js

```diff
@@ -26,7 +26,7 @@ function errorMessage (er) {
     case 'EACCES':
     case 'EPERM':
       short.push(['', er])
-      detail.push(['', ['\nPlease try running this command again as root/Administrator.'
+      detail.push(['', ['\nPlease adjust file ownership or permissions -- or run this command again as a different user.'
```

Comment:
> run this command again as a different user

This needs to be worded better – as such, it turns a clear, explicit action item (which a number of people disagree with) into something confusingly vague.